src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -1045,3 +1045,24 @@ function dorogovtsev_mendes(n::Integer; seed::Int=-1)
     end
     return g
 end
+
+"""
+    random_orientation_dag(n)
+
+Generate a random oriented acyclical digraph. The function takes in a simple/
+graph as an argument. 
+"""
+function random_orientation_dag(g::SimpleGraph{T}) where T <: Integer
```

Comment:
this function should be made reproducible by passing a last argument `rng` with default
argument `Random.GLOBAL_RNG`
Also, could there be an additional probability cutoff of generating the edge?

Reply:
We currently assumed that the magnitude of the edges are given to us, and that to the function, the graph weights are deterministic from the pov of the function. A further addition could be that we generate a graph immediately from # of vertices and the probability of an edge occuring.